README.md

```diff
@@ -120,6 +120,19 @@ letsencrypt::certonly { 'foo':
 }
 ```
 
+#### Nginx authenticator
+
+To request a certificate for $::fqdn with the `certonly` installer and the `nginx` authenticator, temporarily opening firewall for HTTP, with an already running Nginx, and reloading Nginx afterwards:
+
+```puppet
+letsencrypt::certonly { $::fqdn:
```

Comment:
please don't use the legacy topscope syntax to access facts. instead you can use the facts hash:
```suggestion
letsencrypt::certonly { $facts['fqdn']:
```